test/caption-stream-test.js

```diff
@@ -9,17 +9,38 @@
     }
   });
 
-  QUnit.skip('parses SEIs larger than 255 bytes', function() {
-    var captions = [];
-    captionStream.on('data', function(caption) {
-      captions.push(caption);
-    });
+  test('parses SEIs larger than 255 bytes', function() {
```

Comment:
Should this test description be updated? Looks like we're now also testing multi-message SEI packets.